File: app/src/main/java/com/example/sheduleapp_v5/models/LessonIndex.java
package com.example.sheduleapp_v5.models;

import java.util.List;

public class LessonIndex {
    private String lessonStartTime;
    private String lessonEndTime;
    private List<LessonItem> items; // Список уроков в этом индексе

    public String getLessonStartTime() {
        return lessonStartTime;
    }

    public String getLessonEndTime() {
        return lessonEndTime;
    }

    public List<LessonItem> getItems() {
        return items;
    }
}

File: app/src/main/java/com/example/irkpo_management/models/LessonIndex.kt
package com.example.irkpo_management.models

class LessonIndex {
    @JvmField
    val lessonStartTime: String? = null
    @JvmField
    val lessonEndTime: String? = null
    @JvmField
    val items: MutableList<LessonItem?>? = null // Список уроков в этом индексе
}
